src/Pelagos/Bundle/AppBundle/Resources/views/Tombstone/index.html.twig

```diff
@@ -0,0 +1,33 @@
+{% extends "PelagosAppBundle:template:UI.html.twig" %}
+
+{% block head %}
+    {{ parent() }}
+{% endblock %}
+
+{% block body %}
+    {% if dataset %}
+        <h2>
+            The dataset {{ dataset.udi }} is not available yet for download.
```

Comment:
"not available yet" or "not yet available" ?

Reply:
ticket says 'not available yet'
https://english.stackexchange.com/questions/147431/is-it-more-correct-to-say-you-have-not-yet-actioned-or-you-have-not-action